files/templates/submission.html

```diff
@@ -477,7 +477,7 @@ <h1 id="post-title" class="card-title post-title text-left mb-md-3">
 				<input type="hidden" name="formkey" value="{{v.formkey}}">
 				<input type="hidden" name="parent_fullname" value="t2_{{p.id}}">
 				<input autocomplete="off" id="reply-form-submission-{{p.fullname}}" type="hidden" name="submission" value="{{p.id}}">
-				<textarea required autocomplete="off" {% if not (p and p.id in ADMIGGERS) %}{% if v.longpost %}minlength="280"{% elif v.bird %}maxlength="140"{% endif %}{% endif %} minlength="1" maxlength="10000" oninput="markdown('reply-form-body-{{p.fullname}}', 'form-preview-{{p.id}}');charLimit('reply-form-body-{{p.fullname}}','charcount-reply')" id="reply-form-body-{{p.fullname}}" data-fullname="{{p.fullname}}" class="comment-box form-control rounded" id="comment-form" name="body" form="reply-to-{{p.fullname}}" aria-label="With textarea" placeholder="Add your comment..." rows="3"></textarea>
+				<textarea required autocomplete="off" {% if not (p and p.id in ADMINISTRATORS) %}{% if v.longpost %}minlength="280"{% elif v.bird %}maxlength="140"{% endif %}{% endif %} minlength="1" maxlength="10000" oninput="markdown('reply-form-body-{{p.fullname}}', 'form-preview-{{p.id}}');charLimit('reply-form-body-{{p.fullname}}','charcount-reply')" id="reply-form-body-{{p.fullname}}" data-fullname="{{p.fullname}}" class="comment-box form-control rounded" id="comment-form" name="body" form="reply-to-{{p.fullname}}" aria-label="With textarea" placeholder="Add your comment..." rows="3"></textarea>
```

Comment:
Did you change this everywhere? if it's not changed at the render call in python, won't this break?

Reply:
That's weird, I could've sworn I checked that one. I do not understand how I'm having diff losses on *just this repo*.

Anyway, fixed, and double-checked that there's no remaining references to it.